DatadogSessionReplay/SRSnapshotTests/SRSnapshotTests.xcodeproj/xcshareddata/xcschemes/SRSnapshotTests.xcscheme

```diff
@@ -27,7 +27,17 @@
       selectedDebuggerIdentifier = "Xcode.DebuggerFoundation.Debugger.LLDB"
       selectedLauncherIdentifier = "Xcode.DebuggerFoundation.Launcher.LLDB"
       shouldUseLaunchSchemeArgsEnv = "YES"
-      codeCoverageEnabled = "YES">
+      codeCoverageEnabled = "YES"
+      onlyGenerateCoverageForSpecifiedTargets = "YES">
+      <CodeCoverageTargets>
+         <BuildableReference
+            BuildableIdentifier = "primary"
+            BlueprintIdentifier = "DatadogSessionReplay"
+            BuildableName = "DatadogSessionReplay"
+            BlueprintName = "DatadogSessionReplay"
+            ReferencedContainer = "container:..">
+         </BuildableReference>
+      </CodeCoverageTargets>
```

Comment:
Before, it was gathering test coverage for all targets (e.g. including `TestUtilities`) which made no sense. Now it measures `DatadogSessionReplay` only. Current score is 85% (with units + snapshot tests).